JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -91,185 +99,274 @@
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/PlantCO2Emission"/>
         <rdfs:comment>A relationship between a factory and its overall annual CO2 emission.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
-        <rdfs:label>has plant CO2 emission</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>plant CO2 emission</rdfs:label>
     </owl:ObjectProperty>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#hasPowerConsumption -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasPowerConsumption -->
 
-    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany#hasPowerConsumption">
-        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany#Company"/>
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasPowerConsumption">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Company"/>
         <rdfs:range rdf:resource="http://www.theworldavatar.com/ontology/ontopowsys/PowSysFunction.owl#PowerConsumption"/>
         <rdfs:comment>A relationship between an entity and its annual power consumption.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
-        <rdfs:label>has power consumption</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>power consumption</rdfs:label>
     </owl:ObjectProperty>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#hasSpecificEnergyConsumption -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasSpecificEnergyConsumption -->
+
 
-    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany#hasSpecificEnergyConsumption">
-        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany#Factory"/>
-        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany#SpecificEnergyConsumption"/>
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasSpecificEnergyConsumption">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/SpecificEnergyConsumption"/>
         <rdfs:comment>A relationship between a chemical plant and its design capacity.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
-        <rdfs:label>has specific energy consumption</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>specific energy consumption</rdfs:label>
```

Comment:
Some labels are capitalised, some are not. Could you standardise the naming?

Reply:
Done.